### PR TITLE
Optimized digital pathology pipeline

### PR DESCRIPTION
**Pull request by drbeh**
### Description
This PR adds an optimized digital pathology pipeline for metastasis detection. The pipeline uses the latest cucim release (21.8.2), which is scheduled for today for official release.

### Status
**Ready**

### CODE REVIEW

**Comment by drbeh** [comment]
Hi @wyli, I cannot request for reviewers here and it's very inconvenient. I'd appreciate if you can check my permission for this repo. Thanks

File: pathology/tumor_detection/torch/camelyon_train_evaluate_pytorch_gpu.py [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+    )
+    valid_dataset = PatchWSIDataset(
+        data=valid_json_info_list,
+        region_size=cfg["region_size"],
+        grid_shape=cfg["grid_shape"],
+        patch_size=cfg["patch_size"],
+        transform=preprocess_cpu_valid,
+        image_reader_name="openslide" if cfg["use_openslide"] else "cuCIM",
+    )
+    # _________________________________________________________________________
```
Comment (Nic-Ma) [inline pathology/tumor_detection/torch/camelyon_train_evaluate_pytorch_gpu.py:294]: I would suggest to remove the "___________________" comments in the code.

Thansk.

**Comment by wyli** [comment]
> Hi @wyli, I cannot request for reviewers here and it's very inconvenient. I'd appreciate if you can check my permission for this repo. Thanks

done, thanks!

File: pathology/tumor_detection/torch/camelyon_train_evaluate_pytorch_gpu.py (hunk 1 above)
Comment (drbeh) [reply pathology/tumor_detection/torch/camelyon_train_evaluate_pytorch_gpu.py:294]: Sure, I'll do.

**Comment by drbeh** [comment]
@Nic-Ma I also added the NVTX annotated version. Please let me know if you have any other comments. Thanks.

**Review by wyli — APPROVED** [review]

**Pull request merged**